### PR TITLE
Add TFSec and Terraform Docs to Terraform features

### PR DESCRIPTION
**Pull request by acesyde**
Hi here,

This is a PR to add 2 new feature sets to Terraform

- TFSec
- Terraform Docs

Have a nice day

### CODE REVIEW

**Comment by acesyde** [comment]
@microsoft-github-policy-service agree

File: src/terraform/devcontainer-feature.json [hunk 1]
```diff
@@ -32,6 +32,22 @@
...
+            ],
+            "default": "latest",
+            "description": "TFSec version"
+        },
+        "terraform-docs": {
+            "type": "string",
+            "proposals": [
+                "latest"
+            ],
+            "default": "latest",
```
Comment (samruddhikhandale) [inline src/terraform/devcontainer-feature.json:49]: If you decide to covert these two options to `boolean`, then I wonder if the defaults should be `false` 🤔 

Deferring this decision to @craiglpeters

File: src/terraform/install.sh [hunk 2]
```diff
@@ -212,6 +218,42 @@ if [ "${TERRAGRUNT_VERSION}" != "none" ]; then
     mv -f /tmp/tf-downloads/${terragrunt_filename} /usr/local/bin/terragrunt
 fi
 
+if [ "${TFSEC_VERSION}" != "none" ]; then
+    tfsec_filename="tfsec_${TFSEC_VERSION}_linux_${architecture}.tar.gz"
+    echo "Downloading TFSec... ${tfsec_filename}"
```
Comment (samruddhikhandale) [inline src/terraform/install.sh:223]:
```suggestion
    echo "(*) Downloading TFSec... ${tfsec_filename}"
```

File: src/terraform/install.sh [hunk 3]
```diff
@@ -212,6 +218,42 @@ if [ "${TERRAGRUNT_VERSION}" != "none" ]; then
...
+    fi
+    mkdir -p /tmp/tf-downloads/tfsec
+    tar -xzf /tmp/tf-downloads/${tfsec_filename} -C /tmp/tf-downloads/tfsec
+    chmod a+x /tmp/tf-downloads/tfsec/tfsec
+    mv -f /tmp/tf-downloads/tfsec/tfsec /usr/local/bin/tfsec
+fi
+
+if [ "${TERRAFORM_DOCS_VERSION}" != "none" ]; then
+    tfdocs_filename="terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-${architecture}.tar.gz"
+    echo "Downloading Terraform docs... ${tfdocs_filename}"
```
Comment (samruddhikhandale) [inline src/terraform/install.sh:241]:
```suggestion
    echo "(*) Downloading Terraform docs... ${tfdocs_filename}"
```

**Review by samruddhikhandale — COMMENTED** [review]
Thanks for adding these options! ✨ 

Left some comments.

**Comment by acesyde** [comment]
> Thanks for adding these options! ✨
> 
> Left some comments.

Thks for the Review ;)

File: src/terraform/devcontainer-feature.json [hunk 4]
```diff
@@ -32,6 +32,16 @@
             ],
             "default": "latest",
             "description": "Terragrunt version"
+        },
+        "installTFsec": {
+            "type": "boolean",
+            "default": false,
+            "description": "install TF Sec"
```
Comment (samruddhikhandale) [inline src/terraform/devcontainer-feature.json:39]:
```suggestion
            "description": "Install tfsec, a tool to spot potential misconfigurations for your terraform code"
```

File: src/terraform/devcontainer-feature.json [hunk 5]
```diff
@@ -32,6 +32,16 @@
...
+        },
+        "installTerraformDocs": {
+            "type": "boolean",
+            "default": false,
+            "description": "install Terraform Docs"
```
Comment (samruddhikhandale) [inline src/terraform/devcontainer-feature.json:44]:
```suggestion
            "description": "Install terraform-docs, a utility to generate documentation from Terraform modules"
```

**Review by samruddhikhandale — COMMENTED** [review]
Thanks for adding tests! Left some minor comments!

**Pull request merged**